src/components/common/ColonyActionsTable/hooks/useColonyActionsTableColumns.tsx

```diff
@@ -8,11 +8,10 @@ import { defineMessages } from 'react-intl';
 import { useMobile } from '~hooks';
 import { type ActivityFeedColonyAction } from '~hooks/useActivityFeed/types.ts';
 import { type RefetchMotionStates } from '~hooks/useNetworkMotionStates.ts';
-import { MotionState } from '~utils/colonyMotions.ts';
 import { formatText } from '~utils/intl.ts';
-import MotionStateBadge from '~v5/common/Pills/MotionStateBadge/index.ts';
 import TeamBadge from '~v5/common/Pills/TeamBadge/index.ts';
 
+import ActionBadge from '../partials/ActionBadge/index.ts';
```

Comment:
Please import the file directly, and remove the use of the index barrel file

```suggestion
import ActionBadge from '../ActionBadge/ActionBadge.tsx';
```